app/controllers/duties_controller.rb

```diff
@@ -7,6 +7,8 @@ def index
                   .to_date
     @end_date = @start_date.to_date + 6.days
     @places = Place.all
+    @announcements = Announcement.order(created_at: :desc).limit(3)
+    @new_announcements = Announcement.new
```

Comment:
`@new_announcement`?

Reply:
Or actually, just put this two in a private method and call `prepare_announcements` as a private method. Or something. Then you don't need to change the Rubocop thing